jans-cedarling/cedarling/src/init/policy_store.rs

```diff
@@ -61,5 +63,27 @@ pub(crate) fn load_policy_store(
         },
     };
 
+    // Check if there are any trusted issuers in the policy
+    if let Some(trusted_issuers) = &policy.trusted_issuers {
+        for issuer in trusted_issuers {
+            if let Some(metadata) = &issuer.token_metadata {
+                let tokens_with_role_mapping: Vec<TokenKind> = metadata
+                    .iter()
+                    .filter_map(|x| x.role_mapping.as_ref().map(|_| x.kind.clone()))
+                    .collect();
```

Comment:
After deriving "Copy" to the `TokenKind`

```suggestion
                let tokens_with_role_mapping: Vec<TokenKind> = metadata
                    .iter()
                    .filter_map(|x| x.role_mapping.as_ref().map(|_| x.kind))
                    .collect();
```

Reply:
fix here: [24b2088](https://github.com/JanssenProject/jans/pull/9893/commits/24b20883e3c2ab0a544294f3479ac7eb2e72943d)